src/main/java/io/github/frqnny/darkenchanting/client/gui/DarkEnchanterGUI.java

```diff
@@ -219,6 +226,33 @@ public void recalculateEnchantmentCost() {
         this.context.run((world, blockPos) -> {
             int totalExperience = playerInventory.player.totalExperience;
             enchantCost = BookcaseUtils.applyDiscount(XPUtil.getXpCostFromMap(enchantmentsToApply, enchantmentsOnStack), world, blockPos);
+
+            if(BookcaseUtils.getObsidianCount(world, blockPos))
+            {
```

Comment:
please have it like this if () { and not if () (next line) {

like, keep brackets on the same line in all the code, can you fix this please?...